### ISSUE TITLE
have no idea how to use it ,this type of prompt was work in blender gpt but not here

### ISSUE THREAD

**Issue by schxar** [TREE-Ind/UnrealGPT#3]
![image](https://user-images.githubusercontent.com/65155656/229468296-e4e573ca-4c18-48a8-b19c-c366e8e1f2c5.png)

can you give me some Example prompt ?

![Snipaste_2023-04-03_17-25-57](https://user-images.githubusercontent.com/65155656/229469105-7b3b3d5b-0d70-4f0c-9338-a05b2e1490fc.png)

**Comment by Irontaxi** [comment]
Same issue. Appears to do nothing but says success

**Comment by Irontaxi** [comment]
RIP me. 

![image](https://user-images.githubusercontent.com/6070117/229634528-456e3fb6-5250-4eea-9c0a-d912807e4686.png)

seems I have to pay for open AI api key access

**Comment by YinMJ** [comment]
same issue, did you resolve?

**Comment by schxar** [comment]
> same issue, did you resolve?

no, not really, I just rent my friend  api key and use it, work on every other gpt API project and app but this unreal gpt is simply not working, has no idea how to fix this kind of plugin.

**Comment by TREE-Ind** [comment]
New version will require a key still, but installation is as simple as installing a plugin now.

Can do a lot more than add a cube now as well :D
